### PR TITLE
examples: add grpc-typescript

### PR DESCRIPTION
**Pull request by srenatus**
[The docs PR is here](https://github.com/StyraInc/docs/pull/109)

### CODE REVIEW

File: examples/grpc-typescript/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "grpc-typescript",
+  "version": "0.0.1",
+  "description": "Example for using Enterprise OPA's gRPC API from TypeScript",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (anderseknert) [inline examples/grpc-typescript/package.json:8]: This got me curious! What's it for?

File: examples/grpc-typescript/policy.rego [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package my.policy
+import rego.v1
```
Comment (anderseknert) [inline examples/grpc-typescript/policy.rego:2]: Another newline after package here too :)

**Review by anderseknert — APPROVED** [review]

File: examples/grpc-typescript/package.json (hunk 1 above)
Comment (srenatus) [reply examples/grpc-typescript/package.json:8]: oh that's just the default that npm generates and that I hadn't touched 😳

File: examples/grpc-typescript/policy.rego (hunk 2 above)
Comment (srenatus) [reply examples/grpc-typescript/policy.rego:2]: yes I know but I don't want to 🙃

Comment (anderseknert) [reply examples/grpc-typescript/policy.rego:3]: Lol! There are many things to disagree with the formatter about, but this is _not_ one of them 😆

**Pull request merged**